rust/load/src/workloads.rs

```diff
@@ -36,5 +36,55 @@ pub fn all_workloads() -> HashMap<String, Workload> {
                 document: None,
             }),
         ),
+        (
+            "hybrid-fts-vector".to_string(),
+            Workload::Hybrid(vec![
+                (
+                    0.3,
+                    Workload::Get(GetQuery {
+                        limit: Distribution::Constant(10),
+                        metadata: None,
+                        document: Some(DocumentQuery::Raw(serde_json::json!({"$contains": "the"}))),
+                    }),
+                ),
+                (
+                    0.7,
+                    Workload::Query(QueryQuery {
+                        limit: Distribution::Constant(10),
+                        metadata: Some(MetadataQuery::Raw(serde_json::json!({"i1": 1000}))),
+                        document: None,
+                    }),
+                ),
+            ]),
+        ),
+        (
+            "hybrid-fts-md-vector".to_string(),
+            Workload::Hybrid(vec![
+                (
+                    0.5,
+                    Workload::Get(GetQuery {
+                        limit: Distribution::Constant(10),
+                        metadata: None,
+                        document: Some(DocumentQuery::Raw(serde_json::json!({"$contains": "the"}))),
+                    }),
+                ),
+                (
+                    0.25,
+                    Workload::Get(GetQuery {
+                        limit: Distribution::Constant(10),
+                        metadata: Some(MetadataQuery::Raw(serde_json::json!({"i1": 1000}))),
+                        document: None,
+                    }),
+                ),
+                (
+                    0.25,
+                    Workload::Query(QueryQuery {
+                        limit: Distribution::Constant(10),
+                        metadata: Some(MetadataQuery::Raw(serde_json::json!({"i1": 1000}))),
+                        document: None,
+                    }),
+                ),
```

Comment:
Could we make the filter a bit more diverse

Reply:
Sure!  You have 100% programmatic access to it.  These workloads match what Evan spec'd and are why I'm starting with these hard-coded workloads.  Any representable workload is startable from the curl interface.